### ISSUE TITLE
NSInternalInconsistencyException: NSWindow drag regions should only be invalidated on the Main Thread

### ISSUE THREAD

**Issue by ancarda** [tordanik/OSM2World#162]
When I start the latest binary from the website, I get this error:

```
$ ./osm2world.sh --gui
could not read config, ignoring it: 
org.apache.commons.configuration.ConfigurationException: Unable to load the configuration from the URL file:/Users/ancarda/Downloads/OSM2World-latest-bin/texture_config.properties
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by jogamp.opengl.awt.Java2D$2 (file:/Users/ancarda/Downloads/OSM2World-latest-bin/lib/jogl/jogl-all.jar) to field sun.java2d.opengl.OGLUtilities.UNDEFINED
WARNING: Please consider reporting this to the maintainers of jogamp.opengl.awt.Java2D$2
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
2020-02-22 18:52:46.778 java[1102:16234] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'NSWindow drag regions should only be invalidated on the Main Thread!'
*** First throw call stack:
(
	0   CoreFoundation                      0x00007fff377828ab __exceptionPreprocess + 250
	1   libobjc.A.dylib                     0x00007fff6da3c805 objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff377ab30c -[NSException raise] + 9
	3   AppKit                              0x00007fff348f9228 -[NSWindow(NSWindow_Theme) _postWindowNeedsToResetDragMarginsUnlessPostingDisabled] + 310
	4   AppKit                              0x00007fff348e0bc5 -[NSWindow _initContent:styleMask:backing:defer:contentView:] + 1416
	5   AppKit                              0x00007fff348e0637 -[NSWindow initWithContentRect:styleMask:backing:defer:] + 42
	6   libnativewindow_macosx.jnilib       0x00000001325fd0c8 Java_jogamp_nativewindow_macosx_OSXUtil_CreateNSWindow0 + 440
	7   ???                                 0x000000011a06bc90 0x0 + 4731616400
)
libc++abi.dylib: terminating with uncaught exception of type NSException
./osm2world.sh: line 34:  1102 Abort trap: 6           java -Djava.library.path=$lpsolvepath $vmparams -jar OSM2World.jar --config texture_config.properties $@
```

Some system info if it's useful:

```
$ java --version
java 13.0.2 2020-01-14
Java(TM) SE Runtime Environment (build 13.0.2+8)
Java HotSpot(TM) 64-Bit Server VM (build 13.0.2+8, mixed mode, sharing)
```

* OS: 64bit Mac OS X 10.15.3 19D76
* Kernel: x86_64 Darwin 19.3.0

**Comment by tordanik** [comment]
This appears to be an [issue](https://jogamp.org/bugzilla//show_bug.cgi?id=1370) in the JOGL library which is said to have been fixed in [this commit](https://github.com/sgothel/jogl/commit/1c697274a3c1e976bd9c9b089d6583edf4f346ae). Roll-out of the fix is scheduled for the 2.4.0 release of JOGL according to their [roadmap](https://jogamp.com/wiki/index.php/SW_Tracking_Report_Objectives_for_the_release_2.4.0).

From what I'm reading, it seems to be a Mac-specific problem, and judging from the error log on the JOGL bug tracker, it used to only be a warning on older versions of Mac OS X. Upgrading our JOGL dependency will likely resolve the issue once they publish their 2.4.0 release. Unfortunately, I'm not aware of a workaround in the meantime except forgoing the GUI and performing the conversion from the command line, using an older OS version, or a different OS (possibly in a VM). :(

**Comment by ancarda** [comment]
Thank you @tordanik. I'll wait for the next release
